functions/exercise2/script.js

```diff
@@ -16,8 +16,11 @@ function showResult() {
         var isPhrasePalindrome = isPalindrome(phrase);
         var message = isPhrasePalindrome ? "is a palindrome" : "isn't a palindrome";
         resultElement.value = `'${phrase}' ${message}`;
+        document.getElementById("result").classList.add("resultStyle");
```

Comment:
Do you have to apply class resultStyle here? What is the purpose of adding it dynamically while you do not remove it dynamically?